scripts/ingest/quickRestart.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -ex
+
+k8sdir=`dirname ${0}`
+. ${k8sdir}/k8shelper_local.sh
+
+CONFIG_PATH=${k8sdir}/scripts/
+
+export K8SNAMESPACE=$(cd ${CONFIG_PATH} ; ./configuration provisioning.kubernetes_namespace )
+
+wait_timeout=300
+clean=0
+errno=0
+REMOTE_USER=root
+
+##
+#delete the resource definitions
+$k8sdir/delete.sh
+RES=$?
+if [ $RES -ne 0 ]; then
+   echo "Warning with return code $RES: resouces might not be cleaned up completely"
+fi
+
+# Move the data aside
+$k8sdir/clean.sh --quick_move --user "${REMOTE_USER}" --password ""
+RES=$?
+if [ $RES -ne 0 ]; then
+  echo "Warning with return code $RES: data and metadata might not be cleaned up completedly"
+fi
+
+# Wait 30s to catch potential Terminating pods
+sleep 30
```

Comment:
dubious sleep - a loop checking for terminating pods is more effective. If pods get stuck, the loop can terminate after a certain retry period and proceed with ungraceful shutdown below.